### PR TITLE
Client dto

### PR DESCRIPTION
**Pull request by LizaSandul**

### CODE REVIEW

File: src/main/webapp/app/shared/models/full/user-auth-model.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {BaseModel} from "../base-model";
+import {UserInfoModel} from "./user-info-model";
+
+export class UserAuthModel extends BaseModel {
+    email: string;
+    path: string;
```
Comment (savionak) [inline src/main/webapp/app/shared/models/full/user-auth-model.ts]: rename `path` to `pass`

File: src/main/webapp/app/shared/models/full/user-auth-model.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    role: Role;
+    blocked:boolean;
+    removed:boolean;
+    userInfo: UserInfoModel;
+
+    public static empty(): UserAuthModel {
+        return {id: null, email: "", path: null, role: null, blocked:false,removed:false,userInfo:null}
```
Comment (savionak) [inline src/main/webapp/app/shared/models/full/user-auth-model.ts]: set `MANAGER `as default role

File: src/main/webapp/app/shared/models/full/order-model.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+export class OrderModel extends BaseModel {
+    customer: CustomerBriefModel;
+    cost:number;
+    orderDate: Timestamp;
+    status:Status;
+    export:ExportBriefModel;
+
+    public static empty(): OrderModel {
+        return {id: null, customer: null, cost: null, orderDate:null, status: null, export:null}
```
Comment (savionak) [inline src/main/webapp/app/shared/models/full/order-model.ts]: default status - `IN_PROGRESS`

File: src/main/webapp/app/shared/models/full/export-model.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import {Timestamp} from "rxjs/Rx";
+
+export class ExportModel extends BaseModel {
+    order: OrderBriefModel;
+    exportDate: Timestamp;
+    address:string;
+    done: boolean
+
+    public static empty(): ExportModel {
+        return {id: null, order: null, exportDate: null, address:null, done: false}
```
Comment (savionak) [inline src/main/webapp/app/shared/models/full/export-model.ts]: set `exportDate: new Date().getTime()`

File: src/main/webapp/app/shared/models/full/assembly-component-model.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+import {ComponentStoreBriefModel} from "../brief/component-store-brief-model";
+
+
+export class AssemblyComponentModel extends BaseModel {
+    assemblyId: number;
+    component: ComponentStoreBriefModel;
+    count: number;
+
+    public static empty(): AssemblyComponentModel {
+        return {id: null, assemblyId: null, component: null, count: null}
```
Comment (SVss) [inline src/main/webapp/app/shared/models/full/assembly-component-model.ts]: initialize **all numeric fields**(except `id`)  with **0**

**Review by savionak — CHANGES_REQUESTED** [review]
Initialize all **numeric fields**(except `id`) with **0**

Plus use `Reformat Code` feature:

image

**Review by savionak — APPROVED** [review]

**Pull request merged**